modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/saml/filebasedspidpconfigs/management-console-sso-carbonServer.xml

```diff
@@ -0,0 +1,105 @@
+   <keyEncryptionAlgorithmURI>http://www.w3.org/2001/04/xmlenc#rsa-oaep-mgf1p</keyEncryptionAlgorithmURI>
+   <enableSAML2ArtifactBinding>false</enableSAML2ArtifactBinding>
+   <doValidateSignatureInArtifactResolve>false</doValidateSignatureInArtifactResolve>
+   <samlECP>false</samlECP>
+</samlssoServiceProviderDTO>
+]]></inboundConfiguration>
+                <Properties/>
+            </InboundAuthenticationRequestConfig>
+        </InboundAuthenticationRequestConfigs>
+    </InboundAuthenticationConfig>
+    <LocalAndOutBoundAuthenticationConfig>
+        <AuthenticationSteps>
+            <AuthenticationStep>
+                <StepOrder>1</StepOrder>
+                <LocalAuthenticatorConfigs>
+                    <LocalAuthenticatorConfig>
+                        <Name>BasicAuthenticator</Name>
+                        <DisplayName>Username &amp; Password</DisplayName>
+                        <IsEnabled>false</IsEnabled>
+                        <Properties/>
+                    </LocalAuthenticatorConfig>
+                </LocalAuthenticatorConfigs>
+                <FederatedIdentityProviders/>
+                <SubjectStep>false</SubjectStep>
+                <AttributeStep>false</AttributeStep>
+            </AuthenticationStep>
+        </AuthenticationSteps>
+        <AuthenticationType>local</AuthenticationType>
+        <alwaysSendBackAuthenticatedListOfIdPs>false</alwaysSendBackAuthenticatedListOfIdPs>
+        <subjectClaimUri>http://wso2.org/claims/username</subjectClaimUri>
+        <UseTenantDomainInUsername>true</UseTenantDomainInUsername>
+        <UseUserstoreDomainInRoles>true</UseUserstoreDomainInRoles>
+        <UseUserstoreDomainInUsername>false</UseUserstoreDomainInUsername>
+        <SkipConsent>false</SkipConsent>
+        <skipLogoutConsent>false</skipLogoutConsent>
+        <EnableAuthorization>false</EnableAuthorization>
+    </LocalAndOutBoundAuthenticationConfig>
+    <RequestPathAuthenticatorConfigs/>
+    <InboundProvisioningConfig>
+        <ProvisioningUserStore/>
+        <IsProvisioningEnabled>false</IsProvisioningEnabled>
+        <IsDumbModeEnabled>false</IsDumbModeEnabled>
+    </InboundProvisioningConfig>
+    <OutboundProvisioningConfig>
+        <ProvisioningIdentityProviders/>
+    </OutboundProvisioningConfig>
+    <ClaimConfig>
+        <RoleClaimURI/>
+        <LocalClaimDialect>true</LocalClaimDialect>
+        <IdpClaim/>
+        <ClaimMappings/>
+        <AlwaysSendMappedLocalSubjectId>false</AlwaysSendMappedLocalSubjectId>
+        <SPClaimDialects/>
+    </ClaimConfig>
+    <PermissionAndRoleConfig>
+        <Permissions/>
+        <RoleMappings/>
+        <IdpRoles/>
+    </PermissionAndRoleConfig>
+    <IsSaaSApp>true</IsSaaSApp>
+    <ImageUrl/>
+    <AccessUrl/>
+    <IsDiscoverable>false</IsDiscoverable>
+    <TemplateId/>
+    <IsManagementApp>false</IsManagementApp>
+</ServiceProvider>
```

Comment:
keep blank line at the EOF

Reply:
Resolved with 09477f6870bb83ac426be6e2ee0d7e30fa1dbcdc